03_ScienceExamples/DESI/01a_Intro_to_DESI_DR1-Py3.ipynb

```diff
@@ -6,19 +6,19 @@
    "id": "0f95802f",
```

Comment:
<p>"with Python 3 Kernel" made more sense 3-4 years ago when we were still transitioning from Python 2 to Python 3. Now I look at that and say, "aren't all kernels Python 3 at this point"? I would change the title to something like "with Data Lab's standard software".</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astro-datalab/notebooks-latest/pull/266/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Good point. I think the name of the kernel in Data Lab is "Python 3 (ipykernel)" but I can definitely work on improving the title and description to clarify all that it means just a Python 3 environment with the datalab and sparcl clients. The main audience is people who will run this notebook on their local machine and not on DL (e.g., one UofA colleague asked us specifically for this). 

Are you OK with me merging this branch into baw_desi_dr1 and then I can create a new one dedicated to just the Py3 notebook?

Reply:
<p>Yes that's fine.</p>